tests/suite/test_transport_server_tcp_load_balance.py

```diff
@@ -617,5 +623,88 @@ def test_secure_tcp_request_load_balanced(
             endpoint = response.decode()
             print(f"Connected securely to: {endpoint}")
 
+        # for OSS and and Plus with -ssl-dynamic-reload=false, we expect
+        # replacing a secret to trigger a reload
+        count_before_replace = get_reload_count(transport_server_setup.metrics_url)
+        print(f"replacing: {sec_name} in {transport_server_setup.namespace}")
+        replace_secret(kube_apis.v1, sec_name, transport_server_setup.namespace, src_new_sec_yaml)
+        wait_before_test()
+        print(f"waited to {datetime.now().strftime('%m/%d/%Y, %H:%M:%S')}")
+        count_after = get_reload_count(transport_server_setup.metrics_url)
+        reloads = count_after - count_before_replace
+        expected_reloads = 1
+        assert reloads == expected_reloads, f"expected {expected_reloads} reloads, got {reloads}"
+
         self.restore_ts(kube_apis, transport_server_setup)
         delete_items_from_yaml(kube_apis, src_sec_yaml, transport_server_setup.namespace)
+
+
+@pytest.mark.skip_for_nginx_oss
+@pytest.mark.ts
+@pytest.mark.skip_for_loadbalancer
+@pytest.mark.parametrize(
+    "crd_ingress_controller, transport_server_setup",
+    [
+        (
+            {
+                "type": "complete",
+                "extra_args": [
+                    "-global-configuration=nginx-ingress/nginx-configuration",
+                    "-enable-leader-election=false",
+                    "-enable-prometheus-metrics",
+                    "-v=3",
+                ],
+            },
+            {"example": "transport-server-tcp-load-balance"},
+        )
+    ],
+    indirect=True,
+)
+class TestTransportServerTcpLoadBalanceDynamicReload:
+    def test_secure_tcp_request_load_balanced(
```

Comment:
I think we need to change the test name here to better reflect what's being tested, something like:
`test_dynamic_reload()` should be sufficient